daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -1754,7 +1755,7 @@ func (devices *DeviceSet) UnmountDevice(hash string) error {
 	}
 
 	logrus.Debugf("[devmapper] Unmount(%s)", info.mountPath)
-	if err := syscall.Unmount(info.mountPath, syscall.MNT_DETACH); err != nil {
```

Comment:
losing this `MNT_DETACH` is not fine. This lazy `umount` is keeping this from being a blocking operation.